src/superphot_plus/data_generation/alerce.py

```diff
@@ -0,0 +1,175 @@
+"""This script provides functions for importing and manipulating ZTF 
+data from the Alerce API."""
+
+import csv
+import os
+
+from alerce.core import Alerce
+
+alerce = Alerce()
+MIN_PER_FILTER = 5
+
+
+def add_stamp_column(input_filename, output_filename):
+    """Checks whether stamp classifier categorizes each lightcurve in
+    spreadsheet as a supernova-like transient, and adds as additional
+    column.
+
+    Parameters
+    ----------
+    input_filename : str
+        Path to the input CSV file.
+    output_filename : str
+        Path to the output CSV file.
+    """
+    csv_rows = []
+    with open(input_filename, "r", encoding="utf-8") as fn_csv:
+        csv_reader = csv.reader(fn_csv, delimiter=",")
+        next(csv_reader)
+        for row in csv_reader:
+            csv_rows.append(row)
+
+    print("done reading in rows")
+    with open(output_filename, "w+", encoding="utf-8") as new_csv:
+        csv_writer = csv.writer(new_csv, delimiter=",")
+        csv_writer.writerow(["NAME", "PROB", "CLASS", "STAMP"])
+
+    with open(output_filename, "a+", encoding="utf-8") as new_csv:
```

Comment:
I'm not sure why this function does a second file open. Can't it just use the with statement above?

Reply:
Yes, it can. I was trying not to change the methods toooo much since they're not covered by unit tests so I can't be sure I'm not changing behavior. But this seems pretty obvious.